plugins/kontent-ai-plugin/scripts/setup.js

```diff
@@ -79,10 +99,23 @@ const questions = [
     when: !argv.managementApiKey,
     message: "Your Content Management API access token",
   },
+  {
+    name: "cleanProject",
+    when: !argv.cleanProject,
+    message: `Your project should be empty before migrating. Do you want to clear the project ${argv.projectId} (y/n)`,
```

Comment:
```suggestion
    message: `For best results, this script should be ran against an empty Kontent.ai project. Do you want to clear the project before proceeding? (y/n)`,
```

I have some copy suggestions here, and a suggestion to not depend upon `argv.projectId` as that variable will only be defined if the user executes the `yarn setup` script like so: `yarn setup --projectId="some_uuid_here"` which is pretty unlikely given that's not how we instruct them to proceed in the README.